website/public/css/index.styl

```diff
@@ -110,6 +110,12 @@ html, body
 -------------------------------------------------- */
 //move
 
+.modal-subheading {
+  margin: 12px 0 8px;
+  font-size: 24px;
+  font-weight: bold;
+}
```

Comment:
It doesn't look too bad with the extra spacing from the .lead class, and keeping our css simpler will help with maintenance. Let's go with .lead, if that's okay with you, @tylerxo ?